samples/websocket/chatApp/bundle/resources/carbon.product

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
We decided to make this file optional and dynamically generate this file with Dilan's implementation of "[Architecture] [C5] [Carbon-Feature-Plugin] Dynamic Creation of carbon.product via a Template". Can't we apply that as well here?